web/Dockerfile

```diff
@@ -17,7 +17,7 @@ ARG REACT_APP_REDACTION_SELECT_LIMIT
 
 
 ENV NODE_ENV ${NODE_ENV}
-ENV GENERATE_SOURCEMAP false
```

Comment:
just remember to disable this before prod

Reply:
I'm making it configurable like foiform